Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM node:20-alpine AS base
+
+# Stage 1: Install dependencies
+FROM base AS installer
+
+WORKDIR /usr/src/app
+RUN apk add --no-cache libc6-compat
+COPY package.json package-lock.json ./
+COPY prisma ./prisma
```

Comment:
Copy of prisma is not necessery in this stage, please remove and it should be added in the next stage